Assets/MixedRealityToolkit/Providers/UnityInput/MouseDeviceManager.cs

```diff
@@ -27,17 +27,62 @@ public class MouseDeviceManager : BaseInputDeviceManager, IMixedRealityMouseDevi
             IMixedRealityInputSystem inputSystem,
             string name = null,
             uint priority = DefaultPriority,
-            BaseMixedRealityProfile profile = null) : base(registrar, inputSystem, name, priority, profile) { }
+            BaseMixedRealityProfile profile = null) : base(registrar, inputSystem, name, priority, profile)
+        { }
+
+        // Values defining the range of the cursor and wheel speed multipliers
+        private const float MinSpeedMultiplier = 0.1f;
+        private const float MaxSpeedMultiplier = 10.0f;
+
+        private float cursorSpeed = 1.0f;
+
+        /// <inheritdoc />
```

Comment:
please document all public properties

Reply:
the property should be documented on the interface. I will confirm that the doc exists there